examples/react-urql-query/README.md

```diff
@@ -0,0 +1,15 @@
+# Integrating with React
```

Comment:
I think the only general nit I have is that we can make each example start with `with-`, e.g. `with-next` or `with-react`. I think each name is then as short as what it demonstrates, i.e. `with-react-urql-query` is a little long, because `with-react-urql-mutation` is something that wouldn't have to exist, or if it does, would have to be added to `with-react` as a "general" one.

So basically, we'd be following the least amount of code necessary to demonstrate an integration point.

Reply:
Agreed!